Makefile

```diff
@@ -149,8 +149,10 @@ out/sqlite3.bc: sqlite-src/$(SQLITE_AMALGAMATION)
 	# Generate llvm bitcode
 	$(EMCC) $(CFLAGS) -c sqlite-src/$(SQLITE_AMALGAMATION)/sqlite3.c -o $@
 
-out/extension-functions.bc: sqlite-src/$(SQLITE_AMALGAMATION)/$(EXTENSION_FUNCTIONS)
+# Since the extension-functions.c includes other headers in the sqlite_amalgamation, we declare that this depends on more than just extension-functions.c
+out/extension-functions.bc: sqlite-src/$(SQLITE_AMALGAMATION)
```

Comment:
I believe this is a more correct dependency declaration, and has the benefit of saying, "If you build either of these .bc files, you'll need to unpack the whole sqlite src amalgamation folder"